mesecons_noteblock/init.lua

```diff
@@ -61,6 +62,18 @@ mesecon.noteblock_play = function(pos, param2)
 	pos.y = pos.y-1
 	local nodeunder = minetest.get_node(pos).name
 	local soundname = node_sounds[nodeunder]
+	local pitch
```

Comment:
Maybe a more descriptive name:
```suggestion
	local use_pitch
```